src/containers/cambrian/deck.jsx

```diff
@@ -254,14 +288,28 @@ class Deck extends React.Component {
       })
     }
 
-    handleUpdateDeck(event) {
+    handleUpdateDeck() {
```

Comment:
Why remove the event? It is inconsistent with the rest and we might need it. It is the button click event?

Reply:
Is it better to have consistent function declarations or unused variables? 

If it's consistent for you I will return it.